### PR TITLE
[RFC] Add test for :drop 

### PR DESCRIPTION
**Pull request by KillTheMule**
Cf. #4995

### CODE REVIEW

File: test/functional/ui/drop_spec.lua [hunk 1]
```diff
@@ -0,0 +1,80 @@
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local clear, feed, execute = helpers.clear, helpers.feed, helpers.execute
+
+describe("'drop'", function()
```
Comment (mhinz) [inline test/functional/ui/drop_spec.lua]: It's not an option. :-> `:drop`, please.

Comment (KillTheMule) [inline test/functional/ui/drop_spec.lua]: Whoopsie, that's what I get from copying over other tests :) Will change, thanks!

File: test/functional/ui/drop_spec.lua [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      [3] = {bold = true},
+    })
+    execute("set laststatus=2")
+  end)
+
+  after_each(function()
+    screen:detach()
+  end)
+
+  it("works like :e when called at the start of a session", function()
```
Comment (mhinz) [inline test/functional/ui/drop_spec.lua]: Nitpick: "sessions" are something else in Vim. Maybe "works like :e when not shown in any window" is better.

**Comment by mhinz** [comment]
LGTM!

File: test/functional/ui/drop_spec.lua (hunk 2 above)
Comment (KillTheMule) [inline test/functional/ui/drop_spec.lua]: Right, I changed the text.

**Comment by justinmk** [comment]
This should go in `ex_cmd` folder.

**Comment by justinmk** [comment]
Or we should put all tests in a single folder, there is not really any purpose to the separate folders...

**Comment by KillTheMule** [comment]
I personally like a bit of organisation into folders, even if it's more of a loose thing. So I put it into `ex_cmds` for now, unless you really want me to put them together.

**Comment by KillTheMule** [comment]
Travis failure unrelated methinks.

**Pull request merged**